source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -97,7 +97,7 @@ void SetupContext(string @namespace)
             {
                 SetupGCloudContext(@namespace);
             }
-            else if(variables.IsSet(SpecialVariables.ClusterUrl)) // Most other auth mechanisms require some manual commands to configure
+            else if (variables.IsSet(SpecialVariables.ClusterUrl)) // Most other auth mechanisms require some manual commands to configure
```

Comment:
At some point we should set up PrettyBot to clean up Calamari automatically - if it's not already.